orchestrator-salt/src/main/resources/salt/salt/nginx/scripts/create-user-facing-cert.sh

```diff
@@ -2,7 +2,51 @@
 
 CERT_ROOT_PATH=/etc/certs-user-facing
 
-certm -d ${CERT_ROOT_PATH} ca generate -o=gateway --overwrite
-certm -d ${CERT_ROOT_PATH} server generate -o=gateway --host hostname --host {{ salt['pillar.get']('gateway:address') }}
+cat > ${CERT_ROOT_PATH}/openssl.cnf <<EOF
```

Comment:
it works this way, but I prefer having these file pushed via `file.managed` instead of a `cat` or `echo`